source/2019-09-20-ember-3-13-released.md

```diff
@@ -0,0 +1,190 @@
+
+## The Octane Preview
+
+The Octane features in 3.13 are stable public APIs with our normal SemVer guarantees. If you build an app using APIs in the Octane preview, it will continue to work in future versions.
+
+So why is this a preview and not the final Octane release?
+
+While these APIs are stable, not all Octane features are compatible with popular addons yet. We are also still updating documentation, and important tools like the Ember Inspector do not yet work with some Octane features.
+
+In Ember 3.14, Octane will be the recommended way to use Ember and new apps will have Octane's optional features enabled by default. The [guides](https://octane-guides-preview.emberjs.com/release/) and [tutorial](https://octane-guides-preview.emberjs.com/release/tutorial/00-part-1/) will show Octane examples, and codemods will be available to help users migrate to Octane.
+
+The Ember tutorial has already been completely rewritten for Octane, and the easiest way to understand the big picture of Octane, especially if you're already an experienced Ember developer, is to [work through the new tutorial](https://octane-guides-preview.emberjs.com/release/tutorial/00-part-1/).
+
+---
+
+You can try out the Octane preview by disabling legacy behavior and specifying the Octane edition in `.ember-cli.js`.
+
+```bash
+ember feature:disable jquery-integration
+ember feature:enable template-only-glimmer-components
+ember feature:disable application-template-wrapper
+```
+
+If you need more information on how to migrate away from these legacy features, check out the [Octane release plan](https://blog.emberjs.com/2019/08/15/octane-release-plan.html) blog post.
+
+To opt in to the Octane preview, add the following lines to the beginning of `.ember-cli.js`.
+
+```js
+// .ember-cli.js
+const { setEdition } = require('@ember/edition-utils');
+
+setEdition('octane');
+```
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.13
+
+Ember.js 3.13 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations. There are five (5) new features, one (1) deprecation, and several bugfixes in this version.
+
+#### New Features (5)
+
+**Tracked Properties and Tracked Property Updates (1 of 5)**
+
+Tracked properties introduce a simpler and more ergonomic system for tracking state change in Ember applications. Tracked properties allow Ember to reduce its API surface area while producing code that is both more intuitive and less error-prone.
+
+Have a look at the [guides] to learn more about tracked properties. For design details, check out [RFC 410](https://github.com/emberjs/rfcs/blob/master/text/0410-tracked-properties.md) and [RFC 478](https://github.com/emberjs/rfcs/blob/master/text/0478-tracked-properties-updates.md).
```

Comment:
```suggestion
Have a look at the [guides] to learn more about tracked properties. For design details, check out [RFC 410](https://github.com/emberjs/rfcs/blob/master/text/0410-tracked-properties.md) and [RFC 478](https://github.com/emberjs/rfcs/blob/master/text/0478-tracked-properties-updates.md).
```
This should link to the API docs for tracked, ideally. If we have to, we can link to https://emberjs.com/editions/octane and say "have a look at the draft guides...."